erts/doc/src/erlang.xml

```diff
@@ -2998,9 +2998,20 @@ os_prompt%</pre>
         <pre>
 > <input>hd([1,2,3,4,5]).</input>
 1</pre>
-        <p>Allowed in guard tests.</p>
+        <p>It works with improper lists.</p>
+        <pre>
+> <input>hd([1,2,3,4,5|improper_end]).</input>
+1</pre>
+       <p>Allowed in guard tests.</p>
         <p>Failure: <c>badarg</c> if <c><anno>List</anno></c> is the empty
           list <c>[]</c>.</p>
+        <pre>
+> <input>hd([]).</input>
+** exception error: bad argument
+     in function  hd/1
+        called as hd([])
+        *** argument 1: not a nonempty list
+</pre>
```

Comment:
I don't see the point of this example. I suggest you remove it.

Reply:
done